lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -241,6 +241,13 @@ namespace Js
             DeferredTypeHandler<InitializeNumberPrototype, DefaultDeferredTypeFilter, true>::GetDefaultInstance());
         numberPrototype = RecyclerNew(recycler, JavascriptNumberObject, TaggedInt::ToVarUnchecked(0), tempDynamicType);
 
+        if (scriptContext->GetConfig()->IsESBigIntEnabled())
+        {
+            tempDynamicType = DynamicType::New(scriptContext, TypeIds_BigIntObject, objectPrototype, nullptr,
+                DeferredTypeHandler<InitializeBigIntPrototype, DefaultDeferredTypeFilter, true>::GetDefaultInstance());
+            bigintPrototype = RecyclerNew(recycler, JavascriptBigIntObject, nullptr, tempDynamicType);
```

Comment:
By default, will bigintPrototype be uninitialized memory? That is, if bigint is disabled, should bigintPrototype be set to null?

Reply:
will add that